ext/mysqli/tests/bug81335.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Bug #81335: Packets out of order after connection timeout
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+
+if (!defined('MYSQLI_STORE_RESULT_COPY_DATA')) die('skip requires mysqlnd');
+
+require_once 'connect.inc';
+if (!$link = @my_mysqli_connect($host, $user, $passwd, $db, $port, $socket)) die("skip cannot connect");
+if (mysqli_get_server_version($link) < 80024) {
+    $link->close();
+    die("skip: Due to many MySQL Server differences, the test requires >= 8.0.24");
+}
+$link->close();
+?>
+--FILE--
+<?php
+
+require_once 'connect.inc';
+mysqli_report(MYSQLI_REPORT_ERROR | MYSQLI_REPORT_STRICT);
+$mysqli = new my_mysqli($host, $user, $passwd, $db, $port, $socket);
+$mysqli->query('SET WAIT_TIMEOUT=1');
+usleep(1000000 * 1.1);
+try {
+    $mysqli->query('SELECT SLEEP(1)');
+} catch(mysqli_sql_exception $e) {
+    echo $e->getMessage();
+    echo "\n";
+    echo $e->getCode();
+}
+$mysqli->close();
```

Comment:
This line is not part of the test, right? Can you remove it please?

Reply:
OK

Reply:
Yes, you can remove it.

Reply:
Got it (●'◡'●)